src/main/kotlin/blackjack/model/Player.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.model
+
+const val WIN_SCORE = 21
+
+class Player(val name: String) {
+    private val _cards = mutableListOf<Card>()
+    val cards: List<Card>
+        get() = _cards.deepCopy()
+    var isAvailable: Boolean = true
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val isAvailable: Boolean
```

Reply:
아직 반영되지 않았어요. 확인 부탁드려요.